sentry-ruby/lib/sentry/transaction.rb

```diff
@@ -136,6 +121,13 @@ def finish(hub: nil)
       hub.capture_event(event)
     end
 
+    protected
+
+    def init_span_recorder(limit = 1000)
```

Comment:
@rhcarvalho I've undone the first commit, so it's back to `transaction.rb` now. I also renamed it to match other SDK's names.